# This message is used to navigator action output 

uint8 STOP=0
uint8 MOVE=1
uint8 FINISH=2
uint8 BATTERYLOW=3

int32 action	# navigator action

